src/python/grpcio/grpc/utilities.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2016, Google Inc.
```

Comment:
What would you think of putting these code elements into the `grpc` package itself?